src/main/kotlin/org/eclipse/tractusx/bdpmcertificatemanagement/config/OpenApiConfiguration.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.eclipse.tractusx.bdpmcertificatemanagement.config
+
+import io.swagger.v3.oas.models.OpenAPI
+import io.swagger.v3.oas.models.info.Info
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+class OpenApiConfiguration {
+    @Bean
+    fun apiInfo(): OpenAPI {
+        return OpenAPI()
+            .info(
+                Info()
+                    .title("BPDM Certificate Management")
+                    .description("Service that manages and shares business partner certificates")
+                    .version("1.0.0")
```

Comment:
If its not too late there is argument to be made to start with version 0.1.0 or 0.0.1 as this application will undergo still drastic changes. a version number of 0.x.x exhibits this as breaking changes between minor versions is expected.

Reply:
Not to late its just start :) keeping it as 0.0.1